freqtrade/freqai/freqai_interface.py

```diff
@@ -75,7 +75,7 @@ def __init__(self, config: Config) -> None:
         if self.keras and self.ft_params.get("DI_threshold", 0):
             self.ft_params["DI_threshold"] = 0
             logger.warning("DI threshold is not configured for Keras models yet. Deactivating.")
-        self.CONV_WIDTH = self.freqai_info.get("conv_width", 2)
+        self.CONV_WIDTH = self.freqai_info.get("conv_width", 1)
```

Comment:
While not relevant to this PR - this setting has 0 impact. conv_width get's the default set in constants - it'll never be empty here if a freqAI configuration is set.